website_and_docs/content/blog/2022/bellatrix-test-automation-framework/index.md

```diff
@@ -0,0 +1,234 @@
+- Extensibility a framework can be extended by the user usually by selective overriding or specialized by user code providing specific functionality.
+- Non-modifiable Framework Code the framework code, in general, is not allowed to be modified. Users can extend the framework, but not modify its code.
+
+### Definition Test Framework ###
+A sub-type of frameworks providing generic code for testing different aspects of our applications- UI, API, security, performance and many other.
+
+Many people refer to Selenium WebDriver as a test automation framework but cannot be more wrong. Here is the official definition - "*Selenium automates browsers. That’s it! What you do with that power is entirely up to you. Primarily, it is for automating web applications for testing purposes, but is certainly not limited to just that. Boring web-based administration tasks can (and should!) be automated as well.*" So, Selenium WebDriver is mostly known as the standard for web automation testing but is actually a tool for controlling browsers that we later use in our software libraries and frameworks to write automated tests.
+A framework can be based on several libraries implementing several APIs, but unlike the normal use of an API, the access to the behavior built into the framework is mediated by extending its content with new classes plugged into the framework itself. Moreover, the overall program flow of control can be out of the control of the caller and in the hands of the framework by inversion of control or a similar mechanism.
+
+#### Definition Test Automation Framework ###
+Various guidelines, coding standards, concepts, processes, practices, project hierarchies, modularity, etc. to support automated testing. The user can follow these guidelines while automating application to take advantages of various productive results.
+
+### BELLATRIX Test Automation Framework ###
+The first version of **[BELLATRIX](https://bellatrix.solutions/)** appeared on 26 December 2017. It was available only for C# initially but written on new back then .NET Core, allowing the framework to be used on all major operating systems (cross-platform).
+One huge advantage of BELLATRIX is cross-technology readiness. It allows you to write tests for different technologies such as Web, Mobile, Desktop, and API. For me, this also includes a similar API. In BELLATRIX, we strive the API for different modules to be as identical as possible. Here by API, I don't mean web API.
+
+### Definition API ###
+API or application programming interface is a set of functions and procedures that allow the creation of applications which access the features or data of an operating system, application, or other service. A good API makes it easier to develop a computer program by providing all the building blocks, which are then put together by the programmer.
+
+###Multiple Test Environments Configuration###
```

Comment:
From this point on, it is hard for me to grasp why BELLATRIX is a good alternative. Wouldn't be better to show a simple "how to get started" so the reader gets interested about it, and after that mention the different features in a bullet list with a short description (like dynamic tests cases, email testing, etc...). 

When going too much into details, one misses the point of how to get started with the framework and what else could it offer me. I hope this makes sense to you.